tests/params/test_params_boundary.py

```diff
@@ -213,3 +216,12 @@ def test_boundary_types():
     assert MassOutflow(massFlowRate=1).type == "MassOutflow"
     with pytest.raises(pd.ValidationError):
         MassOutflow(massFlowRate=-1)
+
+    bc = SubsonicOutflowMach(
+        name="SomeBC",
+        Mach=0.2,
+        turbulenceQuantities=TurbulenceQuantities(turbulentIntensity=0.2, viscosityRatio=10),
+    )
+
+    assert bc.turbulence_quantities.turbulent_intensity == 0.2
+    assert bc.turbulence_quantities.turbulent_viscosity_ratio == 10
```

Comment:
lets extend these tests to cover different scenarios including raised exceptions. You can run `pytest tests --cov-report=html --cov=flow360 && open htmlcov/index.html`
to see test coverage. 
<img width="1070" alt="Screenshot 2023-12-14 at 13 14 31" src="https://github.com/flexcompute/Flow360/assets/83596707/b438d219-ab67-4478-939c-016eb85c4680">

Reply:
Added more tests